docs/visual-basic/programming-guide/concepts/collections.md

```diff
@@ -180,11 +180,11 @@ End Class
   
  Some of the common collection classes are described in this section:  
   
--   @System.Collections.Generic classes  
+-   <xref:System.Collections.Generic> classes  
   
--   @System.Collections.Concurrent classes  
+-   <xref:System.Collections.Concurrent> classes  
   
--   @System.Collections classes  
+-   <xref:System.Collections classes>  
```

Comment:
`>` after Collections, not classes. This doesn't appear in the broken links report, though.

Reply:
Probably because I didn't close it properly. That's why I have a feature request for CATS to find those post publishing